### PR TITLE
Added Save linked file to context menu

### PR DESCRIPTION
**Pull request by ChristiaanLock**
Download files & html pages on links from the context menu. Not enables on images because 'Save image' is already implemented

### CODE REVIEW

**Comment by plrenaudin** [comment]
Nice, completely missed that!

File: js/webviewMenu.js [hunk 1]
```diff
@@ -65,6 +65,19 @@ var webviewMenu = {
 				}));
 			}
 
+			if(!IPCdata.image) {
+				menu.append(new MenuItem({
+					type: "separator"
+				}));
+
+				menu.append(new MenuItem({
+					label: 'Save linked file',
```
Comment (PalmerAL) [inline js/webviewMenu.js]: Can this be changed to "Save Link As..." to match other browsers?

Comment (ChristiaanLock) [inline js/webviewMenu.js]: Done!

**Comment by PalmerAL** [comment]
Can you please squash this into one commit?

**Comment by ChristiaanLock** [comment]
Sure, I'll close this PR and create a new one

**Pull request closed without merging**